packages/cactus-core-api/package.json

```diff
@@ -81,6 +81,7 @@
   },
   "dependencies": {
     "@hyperledger/cactus-common": "1.0.0-rc.3",
-    "axios": "0.21.4"
+    "axios": "0.21.4",
+    "rxjs": "7.3.0"
```

Comment:
@outSH Please see my other comment with the `import type` suggestion for details. TLDR here: you can move `rxjs` to the `devDependency` section to reduce bundle size for the core-api package.

Also: apologies for not spotting this sooner.

Reply:
Sure, done